CMakeLists.txt

```diff
@@ -238,24 +235,59 @@ if (enable_checksums)
   set(SRC ${SRC} checksums.c)
 endif(enable_checksums)
 
-add_library(gc-lib STATIC ${SRC})
-set_target_properties(gc-lib PROPERTIES
-                      COMPILE_DEFINITIONS GC_NOT_DLL)
-#TODO TARGET_LINK_LIBRARIES(...  ...  ${LIBS})
-
-add_library(gcmt-lib STATIC ${SRC})
-set_target_properties(gcmt-lib PROPERTIES
-                      COMPILE_DEFINITIONS GC_NOT_DLL)
+# Create target
+add_library(gc ${SRC})
```

Comment:
Usually, in modern CMake, the creation of the target would be near the top of the file.

That way, definitions can be added with `target_add_definitions()` to specific targets instead of `add_definitions()` which is global.

Reply:
OK, please add a commit or PR.

Reply:
Will do in a separate PR after this one is finished